ts/screens/modal/SystemOffModal.tsx

```diff
@@ -3,32 +3,23 @@
  * not work properly. This is due to avoid user tries to access features or services potentially can't work
  * as expected
  */
-import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
-import * as O from "fp-ts/lib/Option";
-import { pipe } from "fp-ts/lib/function";
 import * as React from "react";
 import { Modal } from "react-native";
 import { useSelector } from "react-redux";
 import _ from "lodash";
 import { OperationResultScreenContent } from "../../components/screens/OperationResultScreenContent";
 import I18n from "../../i18n";
-import { backendStatusSelector } from "../../store/reducers/backendStatus";
+import { backendInfoMessageSelector } from "../../store/reducers/backendStatus/backendInfo";
 import { getFullLocale } from "../../utils/locale";
 const SystemOffModal = () => {
-  const backendStatus = useSelector(backendStatusSelector, _.isEqual);
+  const backendInfoMessage = useSelector(backendInfoMessageSelector);
```

Comment:
Removing the `_.isEqual` comparison function may change the desired behaviour. Are we sure about this?

Reply:
ditto